tests/test_vec_normalize.py

```diff
@@ -244,7 +249,7 @@ def test_obs_rms_vec_normalize():
     assert np.allclose(env.ret_rms.mean, 5.688, atol=1e-3)
 
 
-@pytest.mark.parametrize("make_env", [make_env, make_dict_env])
+@pytest.mark.parametrize("make_env", [make_env, make_dict_env, make_image_env])
```

Comment:
what is the difference with `test_save_load_vecnormalized_image` ? (in the sense, do we need the two tests or does `test_save_load_vecnormalized_image` cover the main use cases already?)

Reply:
it seems that we could remove `test_save_load_vecnormalized_image` in favor of that one? (or the error from the issue won't be raised?)

Reply:
The difference is that the error is raised in `model.load` in `test_save_load_vecnormalized_image`. Here, is it raised when calling `check_vec_norm_equal`. I agree that we could remove `test_save_load_vecnormalized_image` in favor of this one.

Reply:
> or the error from the issue won't be raised?

It would be, since the check (previously checking space equality, now checking shape equality) is in `VecNormalize.set_env` which is called by `VecNormalize.load`.